runtime/compiler/CMakeLists.txt

```diff
@@ -115,6 +115,28 @@ if(JITSERVER_SUPPORT)
 	)
 
 	add_custom_target(j9jit_proto DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.h ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.cpp)
+
+	if(JITSERVER_ENABLE_SSL)
+		add_definitions(-DJITSERVER_ENABLE_SSL)
+
+		if(OPENSSL_CFLAGS)
```

Comment:
The value of `OPENSSL_CFLAGS` doesn't appear to be used here: why does it need to be checked?

Reply:
`OPENSSL_CFLAGS` is used in `build/toolcfg/gnu/common.mk`: 
```
ifeq (true, $(ADD_OPENSSL_CFLAGS))
                C_INCLUDES+=$(subst -I,,$(OPENSSL_CFLAGS))
                CXX_INCLUDES+=$(subst -I,,$(OPENSSL_CFLAGS))
endif
```

Reply:
The reference in `common.mk` doesn't count: that file isn't used with cmake.